tests/test_svg.py

```diff
@@ -921,4 +922,339 @@ def test_polygon_shape(self):
         self.assertEqual(list(spad.x1), [20.0, 30.0, 10.0])
         self.assertEqual(list(spad.y1), [20.0, 10.0, 10.0])
 
+
+class SvgFileTC(SvgParserTB):
+    """
+    Test loading SVG files.
+    """
+
+    TESTDIR = os.path.abspath(os.path.dirname(__file__))
+    DATADIR = os.path.join(TESTDIR, "data")
+
+    SVG_TEST_DATA = {
+        'android.svg': {
+            'spads_count': 3,
+            'cpads_count': 5,
+            'total_segments': 14,
```

Comment:
The number like `spads_count`, `cpads_count`, `total_segments` are highly related to the implementation, and they should keep the same value while we refactor the SVG parser in C++.